services/brig/src/Brig/Provider/DB.hs

```diff
@@ -339,33 +361,75 @@ updateServiceConn pid sid url tokens keys enabled = retry x5 $ batch $ do
     cqlEnabled :: PrepQuery W (Bool, ProviderId, ServiceId) ()
     cqlEnabled = "UPDATE service SET enabled = ? WHERE provider = ? AND id = ?"
 
+--------------------------------------------------------------------------------
+-- Service "Indexes" (tag and prefix)
+
+insertServiceIndexes :: MonadClient m
+                     => ProviderId
+                     -> ServiceId
+                     -> Name
+                     -> Range 0 3 (Set.Set ServiceTag)
```

Comment:
How about a type alias for `Range 0 3 (Set.Set ServiceTag)`?

Reply:
👍